### ISSUE TITLE
Newline parsing isn't working for me.

### ISSUE THREAD

**Issue by angarc** [linuswillner/react-console-emulator#632]
I'm using version 4.0.1.

I'm making a request to my server. The response i'm getting I'm outputting to the console via `console.log()` and this is what I'm seeing:

```
Object { output: "hello world\nhello world\nhello world\n" }
```

I outputting this output message to the terminal, but this is what it looks like:

```
hello world hello world hello world 
```

I even explicitly added the `noNewlineParsing={false}` option, but no luck.

Any ideas as to how I can go about fixing this?

**Comment by linuswillner** [comment]
That is definitely odd. Could you show me the code you're using to fetch the data and sending it to the terminal?

**Comment by angarc** [comment]
Sure! Here is the whole thing

```javascript
commands = {
    python: {
      fn: (arg1) => {
        const terminal = this.terminal.current
        if (arg1 === undefined) {
          return "Sorry, we don't support running python directly in the browser.\n" +
            "At any given point, you can only run the script of the file you're working on."
        }

        if (arg1 !== this.props.problemTitle) {
          return `File "${arg1}" non found.\n`
        }
        // send api request
        const code = this.props.code
        const problemId = this.props.problemId
        axios({
          method: 'get',
          url: `${URI}api/run/${problemId}`,
          params: {
            code
          }
        }).then((response) => {
          console.log(response.data)
          terminal.pushToStdout(response.data.output)
        }).catch((error) => {
          console.log(error)
        });
        return ""
      }
    }
  }
```

If you're wondering what's in the `response.data.output`, it's this:

```
{ output: "hello world\nhello world\nhello world\n" }
```

**Comment by linuswillner** [comment]
Hi, thanks for the report, I'll be investigating this in detail ASAP.

**Comment by adrientremblay** [comment]
Doesn't work for me either :(

**Comment by linuswillner** [comment]
Fixed in 12110e8886a1024178df4b161d6c4039871b28e5. Will be pushed out with the next version.
